### PR TITLE
fix pfp and date issue

### PR DESCRIPTION
**Pull request by elimelt**
Author: Elijah

## What changes were made?

See issues

### CODE REVIEW

File: src/components/MemberProfile.tsx [hunk 1]
```diff
@@ -58,7 +58,7 @@ const MemberProfile: React.FC = () => {
         <Stack direction={{ base: 'column', md: 'row' }} spacing={4}>
           <Avatar
             name={resolveName(member)}
-            src={member.profilePictureUrl || member.username}
```
Comment (ShawnCollinge) [inline src/components/MemberProfile.tsx:61]: What if they do'nt have a profile picture?

**Review by ShawnCollinge — APPROVED** [review]
LGTM other than 1 question

File: src/components/MemberProfile.tsx (hunk 1 above)
Comment (elimelt) [reply src/components/MemberProfile.tsx:61]: Avatar uses their name's initials as a fallback

Comment (elimelt) [reply src/components/MemberProfile.tsx:61]: I was just being silly when I originally did `||`

**Pull request merged**